data/json/mapgen/mansion_boarded.json

```diff
@@ -334,17 +343,18 @@
         "~": { "item": "clutter_bathroom", "chance": 10 },
         ")": { "item": "bed", "chance": 40 },
         "R": { "item": "mansion_bookcase", "chance": 100 },
-        "6": { "item": "panic_room", "chance": 65, "repeat": [ 2, 6 ] },
+        "6": { "item": "panic_room", "chance": 75, "repeat": [ 2, 6 ] },
         "I": { "item": "nightstand", "chance": 40 },
         "d": { "item": "dresser_stack", "chance": 100 }
       },
       "place_loot": [
         { "group": "corpse_mansion", "x": [ 10, 12 ], "y": [ 5, 7 ], "chance": 50, "repeat": [ 0, 2 ] },
         { "item": "television", "x": 12, "y": 17, "chance": 100 }
       ],
+      "//": "Flavor spawn group for panic rooms with a different configuration of monsters for better story-telling.",
       "place_monster": [
-        { "group": "GROUP_ZOMBIE", "x": [ 0, 23 ], "y": [ 12, 23 ], "chance": 35, "repeat": [ 1, 4 ] },
-        { "group": "GROUP_ZOMBIE", "x": [ 11, 13 ], "y": [ 6, 10 ], "chance": 65, "repeat": [ 1, 2 ] }
+        { "group": "GROUP_VANILLA_ONLY_HUMANS", "x": [ 0, 23 ], "y": [ 12, 23 ], "chance": 35, "repeat": [ 2, 3 ] },
+        { "group": "GROUP_FANCY_PANICROOM", "x": [ 11, 13 ], "y": [ 6, 10 ], "chance": 65, "repeat": [ 1, 2 ] }
```

Comment:
This variant of the mansion still spawns outside of the scenario until I parameterise it so adding the nonsense ferals back here isn't ok

Reply:
I had forgotten that, I will change it.

Reply:
Once I do that all the scenario spawns can be placed normally I just decided doing it with eocs was much less disruptive to have to backport for 0.H